src/cljs/rems/administration/create_resource.cljs

```diff
@@ -85,34 +90,58 @@
   {:get-form ::form
    :update-form ::set-form-field})
 
+(def ^:private organization-dropdown-id "organization-dropdown")
 (def ^:private licenses-dropdown-id "licenses-dropdown")
 
+(defn- list-organizations []
+  (let [licenses @(rf/subscribe [::licenses])]
+    (sort (distinct (mapv :organization licenses)))))
+
 (defn- resource-organization-field []
-  (let [readonly (roles/disallow-setting-organization? (:roles @(rf/subscribe [:identity])))]
-    [text-field context {:keys [:organization]
-                         :label (text :t.administration/organization)
-                         :readonly readonly
-                         :placeholder (text :t.administration/organization-placeholder)}]))
+  (let [organizations (list-organizations)
```

Comment:
hmm I think this is wrong. consider the case where an installation doesn't use resource licenses at all. not sure what the fix would be...

Reply:
After discussion, getting the list of organizations from config now.